packages/components/src/internal/components/forms/input/TextInput.tsx

```diff
@@ -145,23 +146,26 @@ export class TextInput extends DisableableInput<TextInputProps, TextInputState>
         }
 
         return (
-            <FormsyInput
-                id={queryColumn.fieldKey}
-                name={queryColumn.fieldKey}
-                placeholder={`Enter ${queryColumn.caption.toLowerCase()}`}
-                required={queryColumn.required}
-                {...inputProps}
-                componentRef={this.textInput}
-                disabled={this.state.isDisabled}
-                help={help}
-                label={this.renderLabel()}
-                labelClassName={showLabel ? labelClassName : 'hide-label'}
-                onChange={this.onChange}
-                step={step}
-                type={type}
-                validations={validations}
-                value={this.getInputValue()}
-            />
+            <>
+                <FormsyInput
+                    id={queryColumn.fieldKey}
+                    name={queryColumn.fieldKey}
+                    placeholder={`Enter ${queryColumn.caption.toLowerCase()}`}
+                    required={queryColumn.required}
+                    {...inputProps}
+                    componentRef={this.textInput}
+                    disabled={this.state.isDisabled}
+                    help={help}
+                    label={this.renderLabel()}
+                    labelClassName={showLabel ? labelClassName : 'hide-label'}
+                    onChange={this.onChange}
+                    step={step}
+                    type={type}
+                    validations={validations}
+                    value={this.getInputValue()}
+                />
+                {includeSpacesWarning && <InternalSpacesWarning value={this.state.inputValue} />}
```

Comment:
If isDisabled, should this still display?

Reply:
Yes, I think we should still display the warning.